### DISCUSSION TITLE
There's an example with shared stores?

### DISCUSSION THREAD

**Discussion by oceangravity** [originjs/vite-plugin-federation#315] [Q&A] [answered]
I'm wondering if a Pinia store can be shared through several modules, in example:

![image](https://user-images.githubusercontent.com/5217583/211693785-e143f614-2612-4889-bc68-2d1168293a64.png)

Is this possible? Thanks!

**Comment by Jiannan-dev** [comment, marked as answer]
#317

**Reply by Jiannan-dev** [reply]
Sorry for the hasty submission, this has an incorrect comment which actually makes sense, don't be misled by this incorrect comment
![图片](https://user-images.githubusercontent.com/70385062/211956106-043f0f48-a165-47b5-81ca-bfe6dc263e2b.png)

**Reply by likang329** [reply]
您好，这个demo现在不好用了，在最新1.3.5的版本中应该怎么处理啊
